packages/data-context/src/actions/CodegenActions.ts

```diff
@@ -19,12 +22,20 @@ export class CodegenActions {
     try {
       const src = await this.ctx.fs.readFile(filePath, 'utf8')
 
-      const result = parseReactComponent(src, reactDocgenResolvers.findAllExportedComponentDefinitions)
+      const linker: Linker = {}
```

Comment:
Shame about mutating the arg here :( I guess this is the babel way of doing things.

Just a style thing, but I wonder if `Map` would be more descriptive. `const linker: Map<string, ...>()`.

Reply:
Map looks nicer!